libs/pilight/protocols/433.92/iwds07.c

```diff
@@ -36,18 +36,24 @@ static int validate(void) {
 	return -1;
 }
 
-static void createMessage(int unit, int battery, int state) {
-	iwds07->message=json_mkobject();
-	json_append_member(iwds07->message, "unit", json_mknumber(unit, 0));
-	json_append_member(iwds07->message, "battery", json_mknumber(battery, 0));
-	if(state == 1) {
-		json_append_member(iwds07->message, "state", json_mkstring("closed"));
-	} else {
-		json_append_member(iwds07->message, "state", json_mkstring("opened"));
+static void createMessage(int unit, int battery, int state, char **message) {
+	int x = 0;
+	x = snprintf((*message), 255, "{\"unit\":%d,", unit);
```

Comment:
Please check your alignment and code style.

Reply:
done